stream/tanium-intel/src/tanium_intel_connector/api_handler.py

```diff
@@ -248,21 +254,12 @@ def create_reputation(self, entity):
                 entity["hashes"] = {}
                 for hash in hashes:
                     entity["hashes"][hash["algorithm"]] = hash["hash"]
-            if "MD5" in entity["hashes"]:
-                entry["md5"] = entity["hashes"]["MD5"]
-                entry["uploadedHash"] = entity["hashes"]["MD5"]
-            else:
-                entry["md5"] = ""
-            if "SHA-1" in entity["hashes"]:
-                entry["sha1"] = entity["hashes"]["SHA-1"]
-                entry["uploadedHash"] = entity["hashes"]["SHA-1"]
-            else:
-                entry["sha1"] = ""
-            if "SHA-256" in entity["hashes"]:
-                entry["sha256"] = entity["hashes"]["SHA-256"]
-                entry["uploadedHash"] = entity["hashes"]["SHA-256"]
-            else:
-                entry["sha256"] = ""
+
+            hashes = entity["hashes"]
+            for hash in ["MD5", "SHA-1", "SHA-256"]:
+                if hash in hashes:
+                    entry[hash] = hashes[hash]
+                    entry["uploadedHash"] = hashes[hash]
```

Comment:
```suggestion
            for hash in ["MD5", "SHA-1", "SHA-256"]:
                if hash in hashes:
                    entry[hash] = hashes[hash]
                    entry["uploadedHash"] = hashes[hash]
                else:
                    entry[hash] = ""
```
Not sure if it's necessary, but empty strings were set before.